### PR TITLE
Mk 6885: Final hydra documentation clean up and review

### PR DESCRIPTION
**Pull request by oduameh**
---

* [ ] CHANGELOG updated or not needed
* [ ] Documentation updated or not needed
* [ ] Haddocks updated or not needed
* [ ] No new TODOs introduced or explained herafter

### CODE REVIEW

File: docs/docs/index.md [hunk 1]
```diff
@@ -2,29 +2,29 @@
 
 This section is a manual for users interested in running a Hydra node.
 
-Hydra node is an implementation of the Hydra Head protocol, designed to enable fast and low-cost transactions by operating off-chain while leveraging the security of the main Cardano blockchain.
+Hydra node is an implementation of the Hydra head protocol, designed to enable fast and low-cost transactions by operating off-chain while leveraging the security of the main Cardano blockchain.
```
Comment (olgahryniuk) [inline docs/docs/index.md]:
```suggestion
Hydra node is an implementation of the Hydra Head protocol, designed to enable fast and low-cost transactions by operating off-chain while leveraging the security of the main Cardano blockchain.
```

File: docs/docs/index.md [hunk 2]
```diff
@@ -2,29 +2,29 @@
...
 
-Decentralized applications (DApps), exchanges, and enterprise-level services can choose to run a Hydra node to participate in forming Hydra Heads – mini-ledgers that process transactions independently and in parallel, significantly enhancing throughput and reducing latency. 
+Decentralized applications (DApps), exchanges, and enterprise-level services can choose to run a Hydra node to participate in forming Hydra heads – mini-ledgers that process transactions independently and in parallel, significantly enhancing throughput and reducing latency. 
 
-The `hydra-node` interfaces with the Cardano blockchain, connects to other `hydra-nodes` on a dedicated overlay network, runs a simplified (coordinated) Hydra Head protocol, and provides an API for clients.
+The `hydra-node` interfaces with the Cardano blockchain, connects to other `hydra-nodes` on a dedicated overlay network, runs a simplified (coordinated) Hydra head protocol, and provides an API for clients.
```
Comment (olgahryniuk) [inline docs/docs/index.md]:
```suggestion
The `hydra-node` interfaces with the Cardano blockchain, connects to other `hydra-nodes` on a dedicated overlay network, runs a simplified (coordinated) Hydra Head protocol, and provides an API for clients.
```

File: docs/docs/index.md [hunk 3]
```diff
@@ -2,29 +2,29 @@
...
 
 :::warning Mainnet availability disclaimer
 
-The Hydra Head protocol version 0.10.0 or newer is compatible with the Cardano
+The Hydra head protocol version 0.10.0 or newer is compatible with the Cardano
```
Comment (olgahryniuk) [inline docs/docs/index.md]:
```suggestion
The Hydra Head protocol version 0.10.0 or newer is compatible with the Cardano
```

File: docs/docs/index.md [hunk 4]
```diff
@@ -2,29 +2,29 @@
...
 mainnet, which means it is possible to run a `hydra-node` on mainnet using real
 funds.
 
-Before running a `hydra-node` to take part in the Hydra Head protocol,
+Before running a `hydra-node` to take part in the Hydra head protocol,
```
Comment (olgahryniuk) [inline docs/docs/index.md]:
```suggestion
Before running a `hydra-node` to take part in the Hydra Head protocol,
```

File: docs/docs/index.md [hunk 5]
```diff
@@ -2,29 +2,29 @@
...
 developers are strongly encouraged to review the [known issues][known-issues] in
 the documentation in order to understand the current limitations and the
 possible consequences.
 
-By using Hydra Head protocol version 0.10.0 or newer, you understand the
+By using Hydra head protocol version 0.10.0 or newer, you understand the
```
Comment (olgahryniuk) [inline docs/docs/index.md]:
```suggestion
By using Hydra Head protocol version 0.10.0 or newer, you understand the
```

File: docs/docs/index.md [hunk 6]
```diff
@@ -2,29 +2,29 @@
...
 protocol is in development and that use of the `hydra-node` on mainnet is
 entirely at your own risk.
 
 You also acknowledge and agree to have an adequate understanding of the risks
-associated with use of the Hydra Head protocol version 0.10.0 or newer and that
+associated with use of the Hydra head protocol version 0.10.0 or newer and that
```
Comment (olgahryniuk) [inline docs/docs/index.md]:
```suggestion
associated with the use of the Hydra Head protocol version 0.10.0 or newer and that
```

File: docs/docs/protocol-overview.md [hunk 7]
```diff
@@ -1,17 +1,17 @@
 # Protocol overview
 
-Hydra is the layer 2 scalability solution for Cardano, designed to increase transaction speed through low latency and high throughput while minimizing transaction costs. [Hydra Head](https://eprint.iacr.org/2020/299.pdf) is the first protocol of the Hydra family that lays the foundation for more advanced deployment scenarios using isomorphic, multi-party state channels. For an introduction to the protocol, refer to these two blog posts: 
+Hydra is the layer 2 scalability solution for Cardano, designed to increase transaction speed through low latency and high throughput while minimizing transaction costs. [Hydra head](https://eprint.iacr.org/2020/299.pdf) is the first protocol of the Hydra family that lays the foundation for more advanced deployment scenarios using isomorphic, multi-party state channels. For an introduction to the protocol, refer to these two blog posts:
```
Comment (olgahryniuk) [inline docs/docs/protocol-overview.md]:
```suggestion
Hydra is the layer 2 scalability solution for Cardano, designed to increase transaction speed through low latency and high throughput while minimizing transaction costs. [Hydra Head](https://eprint.iacr.org/2020/299.pdf) is the first protocol of the Hydra family that lays the foundation for more advanced deployment scenarios using isomorphic, multi-party state channels. For an introduction to the protocol, refer to these two blog posts: 
```

File: docs/docs/protocol-overview.md [hunk 8]
```diff
@@ -1,17 +1,17 @@
...
+In Hydra, a set of parties coordinates to commit a set of UTXOs (owned by the parties) into an off-chain protocol called the head protocol. This UTXO set constitutes the initial head state, which can be evolved by handling smart contracts and transactions among the parties without blockchain interaction, provided all participants behave optimistically. The isomorphic nature of Hydra heads ensures that transaction validation and script execution proceed according to the same rules as on-chain, simplifying engineering and guaranteeing consistency. In case of disputes or termination, the current state of the head is decommitted back to the blockchain, updating the blockchain state to reflect the off-chain protocol evolution. This decommit process is efficient, independent of the number of parties or the size of the head state, and supports incremental commits and decommits, allowing UTXOs to be added or removed from a running head without closing it.
```
Comment (olgahryniuk) [inline docs/docs/protocol-overview.md]:
```suggestion
In Hydra, a set of parties coordinates to commit a set of UTXOs (owned by the parties) into an off-chain protocol called the Head protocol. This UTXO set constitutes the initial head state, which can be evolved by handling smart contracts and transactions among the parties without blockchain interaction, provided all participants behave optimistically. The isomorphic nature of Hydra heads ensures that transaction validation and script execution proceed according to the same rules as on-chain, simplifying engineering and guaranteeing consistency. In case of disputes or termination, the current state of the head is decommitted back to the blockchain, updating the blockchain state to reflect the off-chain protocol evolution. This decommit process is efficient, independent of the number of parties or the size of the head state, and supports incremental commits and decommits, allowing UTXOs to be added or removed from a running head without closing it.
```

File: docs/docs/protocol-overview.md [hunk 9]
```diff
@@ -1,17 +1,17 @@
...
 
 ## Hydra head lifecycle 
 
-There are different flavors and extensions of the Hydra Head protocol, but first, let's discuss the full lifecycle of a basic Hydra head and how it allows for isomorphic state transfer between layer 1 and layer 2.
+There are different flavors and extensions of the Hydra head protocol, but first, let's discuss the full lifecycle of a basic Hydra head and how it allows for isomorphic state transfer between layer 1 and layer 2.
```
Comment (olgahryniuk) [inline docs/docs/protocol-overview.md]:
```suggestion
There are different flavors and extensions of the Hydra Head protocol, but first, let's discuss the full lifecycle of a basic Hydra head and how it allows for isomorphic state transfer between layer 1 and layer 2.
```

File: docs/adr/2021-06-06_002-reactive-core.md [hunk 10]
```diff
@@ -12,9 +12,9 @@ Accepted
 
 ## Context
 
-We are looking for a way of expressing the Hydra Head protocol logic in a Hydra node.
+We are looking for a way of expressing the Hydra head protocol logic in a Hydra node.
```
Comment (olgahryniuk) [inline docs/adr/2021-06-06_002-reactive-core.md]:
```suggestion
We are looking for a way of expressing the Hydra Head protocol logic in a Hydra node.
```

File: docs/adr/2021-06-06_002-reactive-core.md [hunk 11]
```diff
@@ -12,9 +12,9 @@ Accepted
...
 
-The Hydra Head protocol is defined as a _State machine_ in the paper, whose transitions are inputs that come from different sources which can emit outputs to other instances of the state machine or the mainchain. See the [FC2021](https://iohk.io/en/research/library/papers/hydrafast-isomorphic-state-channels/) paper for details
+The Hydra head protocol is defined as a _State machine_ in the paper, whose transitions are inputs that come from different sources which can emit outputs to other instances of the state machine or the mainchain. See the [FC2021](https://iohk.io/en/research/library/papers/hydrafast-isomorphic-state-channels/) paper for details
```
Comment (olgahryniuk) [inline docs/adr/2021-06-06_002-reactive-core.md]:
```suggestion
The Hydra Head protocol is defined as a _state machine_ in the paper, whose transitions are inputs that come from different sources which can emit outputs to other instances of the state machine or the main chain. See the [FC2021](https://iohk.io/en/research/library/papers/hydrafast-isomorphic-state-channels/) paper for details
```

File: docs/adr/2021-06-10_006-network-broadcasts-all-messages.md [hunk 12]
```diff
@@ -12,7 +12,7 @@ Accepted
 
 ## Context
 
-The simplified Head protocol in the [Hydra
+The simplified head protocol in the [Hydra
```
Comment (olgahryniuk) [inline docs/adr/2021-06-10_006-network-broadcasts-all-messages.md]:
```suggestion
The simplified Head protocol in the [Hydra
```

File: docs/benchmarks/index.md [hunk 13]
```diff
@@ -5,7 +5,7 @@ sidebar_position: 1
...
+This section provides up-to-date data regarding the known limitations of the Hydra head on-chain protocol. Cardano transactions and blocks are subject to constraints on transaction size, execution cost, and the number of inputs and outputs. These constraints are determined by network parameters and significantly influence the capabilities of the Hydra head protocol, such as the maximum number of parties that can participate, the amount of UTXOs that can be committed, and the extent to which these UTXOs can be fanned out. As on-chain scripts and transactions are further optimized, and as the underlying Cardano blockchain evolves with expanded parameters and enhanced script execution efficiency, these limitations are expected to evolve.
```
Comment (olgahryniuk) [inline docs/benchmarks/index.md]:
```suggestion
This section provides up-to-date data regarding the known limitations of the Hydra Head on-chain protocol. Cardano transactions and blocks are subject to constraints on transaction size, execution cost, and the number of inputs and outputs. Network parameters determine these constraints and significantly influence the capabilities of the Hydra Head protocol, such as the maximum number of parties that can participate, the amount of UTXOs that can be committed, and the extent to which these UTXOs can be fanned out. As on-chain scripts and transactions are further optimized, and as the underlying Cardano blockchain evolves with expanded parameters and enhanced script execution efficiency, these limitations are expected to evolve.
```

File: docs/docs/dev/scalability.md [hunk 14]
```diff
@@ -23,13 +23,13 @@ Layer 2 solutions, such as state channels, offer an alternative approach by over
 Read more about layer 2 solutions and their types in the next section. 
 
 
-## State channels and the Hydra Head protocol
+## State channels and the Hydra head protocol
```
Comment (olgahryniuk) [inline docs/docs/dev/scalability.md]:
```suggestion
## State channels and the Hydra Head protocol
```

File: docs/docs/dev/scalability.md [hunk 15]
```diff
@@ -23,13 +23,13 @@ Layer 2 solutions, such as state channels, offer an alternative approach by over
...
 
-The Hydra Head protocol is a form of state channel and can be classified as a horizontal scaling solution. Multiple instances can be deployed to offload transactions and enhance the overall scalability of the system. It provides a flexible way to determine the level of decentralization for each instance and offers a mostly frictionless method to transfer state between the main chain and individual Hydra heads due to its isomorphic nature. 
+The Hydra head protocol is a form of state channel and can be classified as a horizontal scaling solution. Multiple instances can be deployed to offload transactions and enhance the overall scalability of the system. It provides a flexible way to determine the level of decentralization for each instance and offers a mostly frictionless method to transfer state between the main chain and individual Hydra heads due to its isomorphic nature.
```
Comment (olgahryniuk) [inline docs/docs/dev/scalability.md]:
```suggestion
The Hydra Head protocol is a form of state channel and can be classified as a horizontal scaling solution. Multiple instances can be deployed to offload transactions and enhance the overall scalability of the system. Due to its isomorphic nature, it provides a flexible way to determine the level of decentralization for each instance and offers a mostly frictionless method to transfer state between the main chain and individual Hydra heads. 
```

File: docs/docs/dev/scalability.md [hunk 16]
```diff
@@ -23,13 +23,13 @@ Layer 2 solutions, such as state channels, offer an alternative approach by over
...
 
 ![State Channel](./state-channel.png)
 
 State channels allow parts of the state from the layer 1 blockchain to be validated off-chain by the concerned parties. Once the computation is complete, the parties return the final agreed-upon state to the layer 1 blockchain. This construction eliminates the 20-second block time limit, enabling state evolution as quickly as the involved parties approve.
 
-In conclusion, the Hydra Head protocol enables operators to balance the trade-off between decentralization, security, and scalability differently from the underlying blockchain, thereby meeting the specific needs of individual applications.
+In conclusion, the Hydra head protocol enables operators to balance the trade-off between decentralization, security, and scalability differently from the underlying blockchain, thereby meeting the specific needs of individual applications.
```
Comment (olgahryniuk) [inline docs/docs/dev/scalability.md]:
```suggestion
In conclusion, the Hydra Head protocol enables operators to balance the trade-off between decentralization, security, and scalability differently from the underlying blockchain, thereby meeting the specific needs of individual applications.
```

File: docs/docs/faqs.md [hunk 17]
```diff
@@ -5,7 +5,7 @@ title: FAQs
 
 <details>
 <summary>What is Hydra?</summary>
-​​Hydra is a family of layer 2 protocols designed to address network scalability capabilities. Hydra Head is the first in this protocol suite, providing the foundation on which to build out further scalability.
+​​Hydra is a family of layer 2 protocols designed to address network scalability capabilities. Hydra head is the first in this protocol suite, providing the foundation on which to build out further scalability.
```
Comment (olgahryniuk) [inline docs/docs/faqs.md]:
```suggestion
​​Hydra is a family of layer 2 protocols designed to address network scalability capabilities. Hydra Head is the first in this protocol suite, providing the foundation on which to build out further scalability.
```

File: docs/docs/faqs.md [hunk 18]
```diff
@@ -19,24 +19,24 @@ Layer 1 solutions provide the foundational infrastructure of a blockchain networ
 </details>
 
 <details>
-<summary>Is Hydra Head secure?</summary>
-Absolutely. Hydra protocols were born out of IOG research, got peer-reviewed, and are implemented using test-driven development. The Hydra Head protocol is a true layer 2 and can fall back directly onto the Cardano layer 1 – hence inheriting the security model of the Cardano blockchain.
+<summary>Is Hydra head secure?</summary>
```
Comment (olgahryniuk) [inline docs/docs/faqs.md]:
```suggestion
<summary>Is the Hydra Head protocol secure?</summary>
```

File: docs/docs/faqs.md [hunk 19]
```diff
@@ -19,24 +19,24 @@ Layer 1 solutions provide the foundational infrastructure of a blockchain networ
...
+Absolutely. Hydra protocols were born out of IOG research, got peer-reviewed, and are implemented using test-driven development. The Hydra head protocol is a true layer 2 and can fall back directly onto the Cardano layer 1 – hence inheriting the security model of the Cardano blockchain.
```
Comment (olgahryniuk) [inline docs/docs/faqs.md]:
```suggestion
Absolutely. Hydra protocols were born out of IOG research, got peer-reviewed, and are implemented using test-driven development. The Hydra Head protocol is a true layer 2 and can fall back directly onto the Cardano layer 1 – hence inheriting the security model of the Cardano blockchain.
```

File: docs/docs/faqs.md [hunk 20]
```diff
@@ -19,24 +19,24 @@ Layer 1 solutions provide the foundational infrastructure of a blockchain networ
...
+This has been previously referenced as a theoretical maximum, but the reality is more nuanced. For a start, with Cardano’s ‘transactions within transactions’ EUTXO model, TPS itself isn’t a useful metric. A Hydra head is like a small community within a larger group. Initially, these communities operate independently. So, adding up their metrics to get a total picture isn't accurate. Since Hydra head uses the EUTXO model, they can process transactions simultaneously without conflicts, especially with good networking, which optimizes resource usage. As the project progresses, we're constantly evaluating its real-world performance in terms of throughput and finality. For more details, read <a href="https://example.com/more-info">this</a> blog post and see the latest benchmarking data <a href="https://example.com/latest-data">here</a>.
```
Comment (olgahryniuk) [inline docs/docs/faqs.md]:
```suggestion
This has been previously referenced as a theoretical maximum, but the reality is more nuanced. For a start, with Cardano’s ‘transactions within transactions’ EUTXO model, TPS itself isn’t a useful metric. A Hydra head is like a small community within a larger group. Initially, these communities operate independently. So, adding up their metrics to get a total picture isn't accurate. Since Hydra heads use the EUTXO model, they can process transactions simultaneously without conflicts, especially with good networking, which optimizes resource usage. As the project progresses, we're constantly evaluating its real-world performance in terms of throughput and finality. For more details, read <a href="https://example.com/more-info">this</a> blog post and see the latest benchmarking data <a href="https://example.com/latest-data">here</a>.
```

File: docs/docs/faqs.md [hunk 21]
```diff
@@ -19,24 +19,24 @@ Layer 1 solutions provide the foundational infrastructure of a blockchain networ
...
 </details>
 
 <details>
-<summary>Can anyone use the Hydra Head protocol?</summary>
-Yes, it's designed to be accessible for developers and end users alike, requiring minimal changes to existing applications to integrate with Hydra Head. However, it is important to note that Hydra is not a network upgrade and it's not like flipping a switch on Cardano to make it fast - instead, applications need to adopt and build on Hydra heads to benefit from it.
+<summary>Can anyone use the Hydra head protocol?</summary>
```
Comment (olgahryniuk) [inline docs/docs/faqs.md]:
```suggestion
<summary>Can anyone use the Hydra Head protocol?</summary>
```

File: docs/docs/faqs.md [hunk 22]
```diff
@@ -19,24 +19,24 @@ Layer 1 solutions provide the foundational infrastructure of a blockchain networ
...
+Yes, it's designed to be accessible for developers and end users alike, requiring minimal changes to existing applications to integrate with Hydra head. However, it is important to note that Hydra is not a network upgrade and it's not like flipping a switch on Cardano to make it fast - instead, applications need to adopt and build on Hydra heads to benefit from it.
```
Comment (olgahryniuk) [inline docs/docs/faqs.md]:
```suggestion
Yes, it's designed to be accessible for developers and end users alike, requiring minimal changes to existing applications to integrate with the protocol. However, it is important to note that Hydra is not a network upgrade, and it's not like flipping a switch on Cardano to make it fast - instead, applications need to adopt and build on Hydra heads to benefit from it.
```

File: docs/docs/faqs.md [hunk 23]
```diff
@@ -19,24 +19,24 @@ Layer 1 solutions provide the foundational infrastructure of a blockchain networ
...
 </details>
 
 <details>
-<summary>When is the Hydra Head protocol a good fit?</summary>
+<summary>When is the Hydra head protocol a good fit?</summary>
```
Comment (olgahryniuk) [inline docs/docs/faqs.md]:
```suggestion
<summary>When is the Hydra Head protocol a good fit?</summary>
```

File: docs/docs/faqs.md [hunk 24]
```diff
@@ -19,24 +19,24 @@ Layer 1 solutions provide the foundational infrastructure of a blockchain networ
...
 
-The Hydra Head protocol is well-suited for situations where a known set of participants know each other well enough to agree on building a network but don’t trust each other enough to manage funds without securing their assets. This security is backed by the possibility of settling disputes on layer 1.
+The Hydra head protocol is well-suited for situations where a known set of participants know each other well enough to agree on building a network but don’t trust each other enough to manage funds without securing their assets. This security is backed by the possibility of settling disputes on layer 1.
```
Comment (olgahryniuk) [inline docs/docs/faqs.md]:
```suggestion
The Hydra Head protocol is well-suited for situations where a known set of participants know each other well enough to agree on building a network but don’t trust each other enough to manage funds without securing their assets. This security is backed by the possibility of settling disputes on layer 1.
```

File: docs/docs/faqs.md [hunk 25]
```diff
@@ -63,7 +63,7 @@ _(\*) It is possible for Hydra nodes to support multiple heads, creating a many-
 </details>
 
 <details>
-<summary>Is the Hydra Head protocol a sidechain?</summary>
+<summary>Is the Hydra head protocol a sidechain?</summary>
```
Comment (olgahryniuk) [inline docs/docs/faqs.md]:
```suggestion
<summary>Is the Hydra Head protocol a sidechain?</summary>
```

File: docs/docs/get-involved.md [hunk 26]
```diff
@@ -4,7 +4,7 @@ title: Get involved
 
 # Get involved
 
-The Hydra Head protocol implementation is an open-source project built on the principles of transparency, collaboration, and community-driven development. As part of the broader Cardano ecosystem, Hydra welcomes [contributions](https://github.com/cardano-scaling/hydra/blob/master/CONTRIBUTING.md) from developers, researchers, and enthusiasts who are committed to advancing decentralized technologies. By participating in the Hydra project, you join a global community dedicated to enhancing Cardano’s scalability, security, and efficiency. 
+The Hydra head protocol implementation is an open-source project built on the principles of transparency, collaboration, and community-driven development. As part of the broader Cardano ecosystem, Hydra welcomes [contributions](https://github.com/cardano-scaling/hydra/blob/master/CONTRIBUTING.md) from developers, researchers, and enthusiasts who are committed to advancing decentralized technologies. By participating in the Hydra project, you join a global community dedicated to enhancing Cardano’s scalability, security, and efficiency.
```
Comment (olgahryniuk) [inline docs/docs/get-involved.md]:
```suggestion
The Hydra Head protocol implementation is an open-source project built on the principles of transparency, collaboration, and community-driven development. As part of the broader Cardano ecosystem, Hydra welcomes [contributions](https://github.com/cardano-scaling/hydra/blob/master/CONTRIBUTING.md) from developers, researchers, and enthusiasts who are committed to advancing decentralized technologies. By participating in the Hydra project, you join a global community dedicated to enhancing Cardano’s scalability, security, and efficiency. 
```

File: docs/docs/tutorial/index.md [hunk 27]
```diff
@@ -1,8 +1,8 @@
 # Open a head on testnet
 
-This tutorial demonstrates how to use `hydra-node` on Cardano's `preprod` testing environment to open a layer 2 state channel between two participants using the Hydra Head protocol. We'll also use [Mithril](https://mithril.network) for fast node bootstrapping.
+This tutorial demonstrates how to use `hydra-node` on Cardano's `preprod` testing environment to open a layer 2 state channel between two participants using the Hydra head protocol. We'll also use [Mithril](https://mithril.network) for fast node bootstrapping.
```
Comment (olgahryniuk) [inline docs/docs/tutorial/index.md]:
```suggestion
This tutorial demonstrates how to use `hydra-node` on Cardano's `pre-production` testing environment to open a layer 2 state channel between two participants using the Hydra Head protocol. We'll also use [Mithril](https://mithril.network) for fast node bootstrapping.
```

File: docs/docs/tutorial/index.md [hunk 28]
```diff
@@ -107,7 +107,7 @@ Please check out each project's GitHub pages for more options.
 
 ## Step 1. Connect to Cardano
 
-The Hydra Head protocol requires a connection to Cardano layer 1 to verify and post protocol transactions in a trustless way. Start by setting up a `cardano-node` on the public testnet. With Mithril, you can skip synchronizing the whole history and get started quickly.
+The Hydra head protocol requires a connection to Cardano layer 1 to verify and post protocol transactions in a trustless way. Start by setting up a `cardano-node` on the public testnet. With Mithril, you can skip synchronizing the whole history and get started quickly.
```
Comment (olgahryniuk) [inline docs/docs/tutorial/index.md]:
```suggestion
The Hydra Head protocol requires a connection to Cardano layer 1 to verify and post protocol transactions in a trustless way. Start by setting up a `cardano-node` on the public testnet. With Mithril, you can skip synchronizing the whole history and get started quickly.
```
